packages/odyssey-design-tokens/src/focus.json

```diff
@@ -1,9 +1,18 @@
 {
   "focus": {
     "outline": {
+      "color": {
+        "primary": { "value": "{color.primary.base.value}" },
+        "danger": { "value": "{color.danger.base.value}" }
+      },
+      "offset": {
+        "base": { "value": "2px" },
+        "tight": { "value": "0" }
+      },
+      "style": { "value": "solid" },
       "width": {
-        "base": { "value": "4px" },
-        "small": { "value": "2px" }
```

Comment:
renaming tokens like this will be a `BREAKING CHANGE` lets make sure to describe our commit correctly to get nice CHANGELOG output at release time